src/app.module.scss

```diff
@@ -1,8 +1,5 @@
 .app {
 	height: 100%;
-	gap: 16px;
-	padding: 32px;
-
```

Comment:
Keep this as it is
and added `padding-bottom : 0` to remove padding from bottom